pennylane/qnodes/qubit.py

```diff
@@ -365,7 +365,14 @@ def metric_tensor(self, args, kwargs=None, *, diag_approx=False, only_construct=
                 # block diagonal approximation
 
                 unitary_op = qml.QubitUnitary(V, wires=list(range(self.num_wires)), do_queue=False)
-                self.device.execute(circuit["queue"] + [unitary_op], [qml.expval(qml.PauliZ(wire)) for wire in list(range(self.device.num_wires))])
+
+                if isinstance(self.device, qml.QubitDevice):
+                    ops = circuit["queue"] + [unitary_op] + [qml.expval(qml.PauliZ(0))]

```

Comment:
```suggestion
                    ops = circuit["queue"] + [unitary_op] + [qml.expval(qml.PauliZ(wire)) for wire in list(range(self.device.num_wires))]
```

Reply:
We would still want to get the computational basis measurement on each wire, correct? (If so, it is odd that no tests failed with the current version.)

Reply:
No, this method just looks at the probabilities :) it ignores the observables.

Reply:
Any observable can be used as long as no rotation occurs (which would affect the probabilities)

Reply:
Oh, I missed it then! Thanks for that